src/main/java/com/ericsson/ei/subscription/SubscriptionHandler.java

```diff
@@ -97,11 +98,16 @@ private void extractConditions(String aggregatedObject,
         try {
             JsonNode subscriptionJson = new ObjectMapper().readTree(
                 subscriptionData);
-            LOGGER.debug("SubscriptionJson : " + subscriptionJson.toString());
-            LOGGER.debug("Aggregated Object : " + aggregatedObject  + " for the event id: " + id);
+			
+            // Remove password  from subscription details and put empty value before logging.
+			JsonNode subscriptoinToDisplay = new ObjectMapper().readTree(subscriptionJson.toString());
+			LOGGER.debug("SubscriptionJson : {}",
+					((ObjectNode) subscriptoinToDisplay).put("password", "").toPrettyString());
```

Comment:
Good to use the json ignore to skip secure fields.

Reply:
As this is the json string, we can not use json ignore property here.

Reply:
I don't see any benefit in printing the whole subscription json in debug level as we are already printing the required info from the subscriptions in next line of debug statements. If we remove this, we can avoid this change/costly operation.  Please comment on this.